src/Core/Settings.h

```diff
@@ -399,6 +399,7 @@ class IColumn;
     \
     /** Obsolete settings that do nothing but left for compatibility reasons. Remove each one after half a year of obsolescence. */ \
     \
+    M(UInt64, max_memory_usage_for_all_queries, 0, "Obsolete. Will be removed after 2020-10-20", 0) \
```

Comment:
Clarification needed.

Reply:
This setting is used in tests

Reply:
All these tests are incorrect and should be changed accordingly.

Reply:
Only tests from Arcadia depend on this setting now. Our colleagues will fix this tests, and then we remove this setting.